CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Vx.x.x (Pre-release)
+- Docs are now available at our new [documentation site](https://pusher.github.io/oauth2_proxy/).
```

Comment:
This should go down with the `Changes since v3.2.0` and follow the pattern set out there, perhaps worth including the original migration PR as the changelog PR as well?